src/contract_wrappers/modules/transfer_manager/count_transfer_manager_wrapper.ts

```diff
@@ -93,6 +102,7 @@ export default class CountTransferManagerWrapper extends ModuleWrapper {
   }
 
   public unpause = async (params: TxParams) => {
+    await this.checkIsPaused();
```

Comment:
worth it define a new private method just for use only once?

Reply:
This exists as a private method inside the wrapper, it needs to call the contract itself to make sure calling the method changes state. For this reason I don’t think I can move to asserts. The only simplification I can think of is moving it to ModuleWrapper and removing altogether from these wrappers. What are you guys thoughts?

Reply:
And it’s used in other cases and useful because some functions check that the contract is not paused before they can act

Reply:
I mean... you can add `assert.assert(await this.paused(), 'Controller not currently paused')` instead. You use `await this.checkIsPaused()` just one time in the wrapper so you can remove the `private checkIsPaused` method.

Reply:
Okay I will add another commit here changing it in any files that only check it once for isPaused